Buildings/Experimental/OpenBuildingControl/ASHRAE/G36/Atomic/EconDamperPositionLimitsMultiZone.mo

```diff
@@ -0,0 +1,312 @@
+  parameter Real conSigFraOutDam(min=conSigMin, max=conSigMax, unit="1")=0.5
+    "Fraction of control loop signal output below which the outdoor air damper limit gets
+    modulated and above which the return air damper limit gets modulated";
+  parameter Real kPDamLim=1 "Gain of damper limit controller";
+  parameter Modelica.SIunits.Time TiDamLim=0.9 "Time constant of damper limit controller integrator block";
+
+  CDL.Interfaces.RealInput VOut_flow(unit="m3/s", quantity="VolumeFlowRate")
+    "Measured outdoor volumetric airflow rate"
+    annotation (Placement(transformation(extent={{-220,150},{-180,190}}),
+        iconTransformation(extent={{-120,70},{-100,90}})));
+  CDL.Interfaces.RealInput VOutMinSet_flow(unit="m3/s", quantity="VolumeFlowRate")
+    "Minimum outdoor volumetric airflow rate setpoint"
+    annotation (Placement(transformation(extent={{-220,200},{-180,240}}),
+        iconTransformation(extent={{-120,40},{-100,60}})));
+  CDL.Interfaces.IntegerInput uOpeMod "AHU operation mode status signal"
+    annotation (Placement(transformation(extent={{-220,-200},{-180,-160}}),
+    iconTransformation(extent={{-120,-60},{-100,-40}})));
+  CDL.Interfaces.IntegerInput uFreProSta "Freeze protection status signal"
+    annotation (Placement(transformation(extent={{-220,-160},{-180,-120}}),
+    iconTransformation(extent={{-120,-90},{-100,-70}})));
+  CDL.Interfaces.BooleanInput uSupFan "Supply fan status signal"
+    annotation (Placement(transformation(extent={{-220,-120},{-180,-80}}),
+        iconTransformation(extent={{-120,-10},{-100,10}})));
+
+  CDL.Interfaces.RealOutput yOutDamPosMin(min=0, max=1, unit="1")
+    "Minimum outdoor air damper position limit"
+    annotation (Placement(transformation(extent={{180,70},{200,90}}),
+    iconTransformation(extent={{100,40},{120,60}})));
+  CDL.Interfaces.RealOutput yOutDamPosMax(min=0, max=1, unit="1")
+    "Maximum outdoor air damper position limit"
+    annotation (Placement(transformation(extent={{180,30},{200,50}}),
+    iconTransformation(extent={{100,60},{120,80}})));
+  CDL.Interfaces.RealOutput yRetDamPosMin(min=0, max=1, unit="1")
+    "Minimum return air damper position limit"
+    annotation (Placement(transformation(extent={{180,-10},{200,10}}),
+    iconTransformation(extent={{100,-10},{120,10}})));
+  CDL.Interfaces.RealOutput yRetDamPosMax(min=0, max=1, unit="1")
+    "Maximum return air damper position limit"
+    annotation (Placement(transformation(extent={{180,-50},{200,-30}}),
+        iconTransformation(extent={{100,-30},{120,-10}})));
+  CDL.Interfaces.RealOutput yRetDamPhyPosMax(min=0, max=1, unit="1")
+    "Physical maximum return air damper position limit. Required as an input for the economizer enable disable sequence"
+    annotation (Placement(transformation(extent={{180,-90},{200,-70}}),
+        iconTransformation(extent={{100,-50},{120,-30}})));
+
+  CDL.Continuous.LimPID damLimController(
+    Ti=TiDamLim,
```

Comment:
Please make these final as otherwise users can overwrite them which is not the intention. And conversely, if we release the code and afterwards make them final, then the code will not be backwards compatible.